Packs/PhishUp/Integrations/PhishUp/PhishUp_test.py

```diff
@@ -0,0 +1,148 @@
+import json
+import io
+
+MOCK_APIKEY = "not"
+MOCK_PARAMS = {
+    "apikey": MOCK_APIKEY
+}
```

Comment:
```suggestion
MOCK_APIKEY = "not"
```

Reply:
@diCagri - Looks like the unit tests are failing because you're passing a `dict`-type to the `token` argument of the client methods and when it's expecting a `str` in order to concat it to the URL. See suggestion.